### PR TITLE
Update ServiceAccountTokenJTI, ServiceAccountTokenPodNodeInfo, ServiceAccountTokenNodeBindingValidation to stable

### PR DESCRIPTION
**Pull request by mengjiao-liu**
### Issue

The upstream K8s PR:
- https://github.com/kubernetes/kubernetes/pull/128169

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -437,10 +436,10 @@ The JSON payload of this token follows a well defined schema - an example payloa
   "exp": 1731613413,
   "iat": 1700077413,
   "iss": "https://kubernetes.default.svc",  # matches the first value passed to the --service-account-issuer flag
-  "jti": "ea28ed49-2e11-4280-9ec5-bc3d1d84661a",  # ServiceAccountTokenJTI feature must be enabled for the claim to be present
+  "jti": "ea28ed49-2e11-4280-9ec5-bc3d1d84661a", 
   "kubernetes.io": {
     "namespace": "kube-system",
-    "node": {  # ServiceAccountTokenPodNodeInfo feature must be enabled for the API server to add this node reference claim
+    "node": {  # for the API server to add this node reference claim
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:442]:
```suggestion
    "node": {
```

**Comment by munnerz** [comment]
Just the one pending comment, otherwise looks great, thanks! ☺️ 

/lgtm

**Review by munnerz — APPROVED** [review]
/lgtm

**Comment by mengjiao-liu** [comment]
ping @tengqm 
for approval

**Pull request merged**